lib/task/import/csvImportTask.class.php

```diff
@@ -1007,7 +1066,7 @@ public function execute($arguments = array(), $options = array())
             $this->log($e->getMessage(), sfLogger::ERR);
```

Comment:
This if block looks like a copy and paste of the code at lines 450 - 474.  I would recommend setting a variable on the first check rather than doing the check twice.